### PR TITLE
Helm chart: add SGX CR

### PR DESCRIPTION
**Pull request by rmstoi**
Signed-off-by: Ruslan Mstoi <ruslan.mstoi@intel.com>

### CODE REVIEW

File: charts/sgx/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: intel-device-plugins-sgx
+description: A Helm chart for Intel Device Plugins SGX
+type: application
+version: 0.1.0
+appVersion: "0.23.0"
```
Comment (mythi) [inline charts/sgx/Chart.yaml:6]:
```suggestion
appVersion: "0.24.0"
```

File: charts/sgx/values.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+image:
+  hub: intel
+  tag: "devel"
+  pullPolicy: IfNotPresent
```
Comment (mythi) [inline charts/sgx/values.yaml:4]: are these used for anything?

**Review by mythi — COMMENTED** [review]
@rmstoi thanks! it works as expected. Two questions/comments.

File: charts/sgx/values.yaml (hunk 2 above)
Comment (rmstoi) [reply charts/sgx/values.yaml]: You are right, these are not needed in the SGX chart.

**Review by mythi — APPROVED** [review]

**Review by bart0sh — APPROVED** [review]
/lgtm

**Pull request merged**